test/Dialect/HW/typedecls.mlir

```diff
@@ -9,3 +9,18 @@ hw.type_scope @__hw_typedecls {
   // CHECK: hw.typedecl @baz, "MY_NAMESPACE_baz" : i8
```

Comment:
Please add a check with nested typedefs.  ```typedecl @x : !hw.struct<a: hw.typealias<...>, b: i1>```

Reply:
I had it in the export verilog test in the other PR, will add it here.